src/core/serialization/mod.rs

```diff
@@ -526,6 +536,43 @@ impl Serializable for CleartextHeader {
     }
 }
 
+impl Serializable for Policy {
+    type Error = Error;
+
+    fn length(&self) -> usize {
+        let bytes = <Vec<u8>>::try_from(self).unwrap();
+        bytes.len() + to_leb128_len(bytes.len())
+
+            // 1 + to_leb128_len(self.last_attribute_value as usize)
+            //     + self
+            //         .dimensions
+            //         .clone()
+            //         .iter()
+            //         .map(|(s, dim)| {
+            //             to_leb128_len(s.len())
+            //                 + s.len()
+            //                 + dim
+            //                     .attributes()
+            //                     .map(|d| to_leb128_len(d.id as usize) + 2)
+            //                     .sum::<usize>()
+            //         })
+            //         .sum::<usize>()
```

Comment:
All commented code is to be removed. you can keep it somewhere in a branch but not in MR.
```suggestion
```